docs/2/installer/lorawan/gateway-installations/outdoor-istation.md

```diff
@@ -4,6 +4,13 @@ title: Kerlink iStation
 description: How to set it up?
 toc: true
 ---
+## Planning your install
+
+{% include image.html width=400 url="/assets/img/LoRaWAN Device pictures/Without text.jpg" description="LoRaWan Technology" %}
```

Comment:
no space, it should be dashes and lower case. Take a look at the other images in the same page